include/NovelRT.h

```diff
@@ -61,8 +61,8 @@
 #include <sndfile.h>
 
 //nethost
-#include <coreclr_delegates.h>
-#include <hostfxr.h>
+#include "NovelRT/DotNet/coreclr_delegates.h"
```

Comment:
I've included these in NovelRT since they're not actually installed by .NET Core, and I don't want to require users download the entire dotnet/runtime repo for a few kilobytes.

AFAIK, the interfaces they provide are fairly stable, so this should be alright (at least for major versions of .NET)